Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Components/Sidebar Header.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Sidebar Header.swift
+//  Mlem
+//
+//  Created by Jake Shirley on 6/21/23.
+//
+
+import SwiftUI
+import CachedAsyncImage
+
+struct CommunitySidebarHeader : View {
+    @State var communityDetails: GetCommunityResponse
+    
+    private func getRelativeTime(date: Date) -> String {
```

Comment:
we should look at moving this type of formatting out into extensions or such for use throughout the app before individual functions start cropping up in different views

Reply:
Moved it to the extensions group!